### ISSUE TITLE
Close the editor window

### ISSUE THREAD

**Issue by anpuhach** [wereturtle/ghostwriter#801]
I actually like the two-panel layout set by default, but sometimes I need more space for the preview window while browsing through the file.
Is it possible to hide it temporarily?

**Comment by wereturtle** [comment]
Absolutely!  There is a button on the lower right that looks like this: `</>`

Alternatively, from the menu bar, you can go to the View menu and uncheck "Preview in HTML".  The keyboard shortcut for this is `CTRL+P`

**Comment by anpuhach** [comment]
Sorry, but you didn't get it.
I want to hide the editor window, not the preview window.

**Comment by wereturtle** [comment]
My bad!  I misunderstood.  Perhaps this can be added as a future feature.
